pdf_command_wx.cpp

```diff
@@ -819,26 +822,57 @@ class pdf_illustration : protected html_interpolator, protected pdf_writer_wx
 
         for(auto const& i : pages_)
             {
-            i->pre_render();
+            try
+                {
+                i->pre_render();
+                }
+            catch(const std::runtime_error& e)
+                {
+                std::ostringstream oss;
+                oss
+                    << e.what()
+                    << '\n'
+                    << "Page name: "
+                    << i->get_name()
+                    ;
+                // Real type of the exception is unknown, so rethrow
```

Comment:
Do we have any classes deriving from `runtime_error`? If not, I'd rather say that we assume that it's ok to rethrow this as a copy because we never use any exceptions of the derived classes.